src/utils.ts

```diff
@@ -4,3 +4,29 @@ export function rexify(expression: string): RegExp {
   });
   return new RegExp(expression);
 }
+
+/**
+ * Check if the number of files changed in a PR matches the expected count
+ * @param octokit - GitHub API client
+ * @param owner - Repository owner
+ * @param repo - Repository name
+ * @param pull_number - Pull request number
+ * @param expectedCount - Expected number of files changed
+ * @returns Promise<boolean> - True if the count matches, false otherwise
+ */
+export async function checkFilesChanged(
```

Comment:
We might need the files that have changed later. It's better if this function returns an array with the files changed.